deps/rabbit/src/rabbit_fifo.erl

```diff
@@ -2126,3 +2163,16 @@ get_priority_from_args(#{args := Args}) ->
     end;
 get_priority_from_args(_) ->
     0.
+
+have_active_consumers_changed(_, false) ->
```

Comment:
This function name doesn't seem to match what it does. The result doesn't answer the question if active consumer have changed.